bgpd/bgp_attr.c

```diff
@@ -1647,8 +1647,8 @@ int bgp_mp_reach_parse(struct bgp_attr_parser_args *args,
 #define BGP_MP_REACH_MIN_SIZE 5
 #define LEN_LEFT	(length - (stream_get_getp(s) - start))
 	if ((length > STREAM_READABLE(s)) || (length < BGP_MP_REACH_MIN_SIZE)) {
-		zlog_info("%s: %s sent invalid length, %lu", __func__,
-			  peer->host, (unsigned long)length);
+		zlog_info("%s: %s sent invalid length, %lu, of MP_REACH NLRI",
```

Comment:
nit: I would use `MP_REACH_NLRI` (without space) to deal with copy/paste/find/replace situations in the future maybe.

Reply:
Got it. Thanks, @ton31337 .